torch/csrc/api/include/torch/nn/options/transformer.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <torch/arg.h>
+#include <torch/csrc/WindowsTorchApiMacro.h>
+#include <torch/types.h>
+#include <torch/enum.h>
+
+#include <torch/nn/modules/container/any.h>
+
+namespace torch {
+namespace nn {
+
+/// Options for the `Transformer` module
+///
+/// Example:
+/// ```
+///
+/// ```
```

Comment:
No example?

Reply:
Ah.....forgot to add it back....was doing cut, but forgot to paste back......will add